experiment/aim.md

```diff
@@ -1 +1 @@
-### Aim of the experiment
+To determine the Optimum Moisture Content (OMC) and maximum dry density of a soil by the standard proctor compaction test.
```

Comment:
1. Should it be 'density of a soil'?
2. Should Proctor be capitalized, since it's a person's name?